### ISSUE TITLE
Missing Icons in slingshot applicaions menu

### ISSUE THREAD

**Issue by CodeVault** [elementary/applications-menu#430]
Application icons are missing in applications-menu when you open menu for first time after reboot or logout.

**Comment by ryonakano** [comment]
Hi, are you using the early-access daily build? I've also been facing (maybe) the same issue recently, like this:

![2020-11-15 20 45 13 の画面録画](https://user-images.githubusercontent.com/26003928/99184025-87f34c00-2783-11eb-9400-d4b99008449e.gif)

If I scroll the view the icons and texts are displayed correctly.

**Comment by CodeVault** [comment]
Hi Ryo, Glad you mailed me. Yes I have been using early access builds for a
few weeks now and I have found a similar issue in applications-menu and
calendar indicator on wingpanel.

**Comment by davidmhewitt** [comment]
Yep, confirmed on 6.0 daily builds.

I'll open a separate issue for the datetime indicator.

**Comment by cassidyjames** [comment]
This is an issue that has been fixed in LibHdy. I think we just need to rebuild the Applications Menu and DateTime indicator.

**Comment by cassidyjames** [comment]
See: https://gitlab.gnome.org/GNOME/libhandy/-/issues/363

**Comment by meisenzahl** [comment]
@exalm clarified on GitLab that the issue should be fixed.

I tried to figure out if the linked MR fixed the issue. Steps I did:

1. Clone, build and install libhandy's `master`
2. Clone, build and install applications-menu's `master`
3. Kill wingpanel and restart it on console

Unfortunately the issue still occurs.

> This is an issue that has been fixed in LibHdy. I think we just need to rebuild the Applications Menu and DateTime indicator.

@cassidyjames was that what you meant by **rebuild**?

**Comment by danrabbit** [comment]
Yeah, it looks like the version of applications-menu in daily was built 11-20 and handy 1.0.2 was built 11-02 so we should have the fix already. So it must not truly be fixed in Hdy

**Comment by meisenzahl** [comment]
I did a time travel in libhandy's git history and was able to find the bug.

| Date | Commit | State |
| ---------- | ---------------------------------------- | ------ |
| 10-08-2020 | 0ee14782358d7a6de7807f535cb60e78c2468f88 | works |
| 11-21-2020 | ddfd9b75e9db1fae9166debb13dc188f68d0f90e | works |
| 11-21-2020 | bdfb9a99e20b5a3139f9828beb95266188af271f | broken |

This fixes the issue upstream: https://gitlab.gnome.org/GNOME/libhandy/-/merge_requests/671

**Comment by meisenzahl** [comment]
The fix has landed on `master`.

**Comment by davidmhewitt** [comment]
@meisenzahl Well done! Fantastic work

**Comment by danrabbit** [comment]
Just an update here, we’re waiting on a release of Hdy. It sounds like we should get 1.0.3 soon with the fix :)

**Comment by meisenzahl** [comment]
Version 1.0.3 of libhandy is now available in elementary 6.0 daily and fixed this.
